README.md

```diff
@@ -93,7 +93,7 @@ REACT_APP_DUBMAP_SERVER=https://o8cid9vnb1.execute-api.us-west-2.amazonaws.com/
   - ![img.png](screenshots/routeNavigation.png)
   - **Room reservation**: On the map homepage, user can hover over the mouse to buildings that are open for room reservation, and an info window will pop up right next to the building with clickable words for user to redirect to the corresponding reservation webpage. Then the user can book a room there. 
   - ![img.png](screenshots/roomReservation.png)
-  - **Dining options**: work in progress
+  - **Dining options**: On the map homepage, user can either enter the course information (e.g. **CSE 403 A**) they are to look for and click the **Show Dining Option** button. The map system will show the navigation route from user's current location to Dining options nearby.
```

Comment:
I don't think this is the expected behavior. Users should just click the get dining option btn to get the information without entering course information. See the use case on dining options below.

Reply:
okay will fix it!